docs/cloud-tutorials/application-management.md

```diff
@@ -39,6 +39,10 @@ When you deploy an application, DBOS Cloud first runs [`npx dbos migrate`](../ap
 It then starts your application.
 After your application is deployed, DBOS Cloud hosts it at this URL, which is also printed by the deploy command:
 
+:::info
+Each application is allocated 1GB of disk space.
```

Comment:
I'm seeing 500MB here: 
https://github.com/dbos-inc/dbos-cloud/blob/733e24f95ec258e06f12151428e42f3fa952c03b/services/applications/deploy.go#L22

Is the code lying?

Reply:
I'm still hung up on the idea of "disk space".  You shouldn't be getting "disk space".  Is it that the application image should be less than 1GB (or 500MB)?

Reply:
> I'm seeing 500MB here: https://github.com/dbos-inc/dbos-cloud/blob/733e24f95ec258e06f12151428e42f3fa952c03b/services/applications/deploy.go#L22
> 
> Is the code lying?

No, you are right, but during the build step, we allow 1GB of disk to accommodate for dev dependencies. So a user could see >500MB from their app. But the deploy will fail if the archive size is more than 500MB. I will update now.*


> I'm still hung up on the idea of "disk space". You shouldn't be getting "disk space". Is it that the application image should be less than 1GB (or 500MB)?

I'll adjust based on Alex comment and say that `Each application can be up to 500MB in size`

Reply:
Also clarified this section